app/src/main/java/org/covidwatch/android/data/CovidExposureInformation.kt

```diff
@@ -16,12 +17,16 @@ data class CovidExposureInformation(
     val id: Int = 0
 ) {
 
-    val howClose: Int
-        get() = when (attenuationValue) {
-            in 0..100 -> R.string.far_exposure_distance
-            in 101..200 -> R.string.close_exposure_distance
-            else -> R.string.near_exposure_distance
-        }
+    @Ignore
+    val howClose = when (attenuationValue) {
```

Comment:
What is this relevant for?

Reply:
[item_exposure_info.xml](https://github.com/covid19risk/covidwatch-android-en/blob/ba927b4560896fc25a6c404f9b536d3ab614cbbe/app/src/main/res/layout/item_exposure_info.xml)

This is code from the `ag-minimal` app but I'm planning to use that screen as a debug screen that controls EN